x-pack/plugins/ml/public/application/components/multi_select_picker/multi_select_picker.tsx

```diff
@@ -100,7 +100,6 @@ export const MultiSelectPicker: FC<{
     <EuiFilterGroup data-test-subj={dataTestSubj}>
       <EuiPopover
```

Comment:
Not something that automated tests will pick, and we do hope to change the default in EUI at some point in the future, but, in the mean time, can we set this popover to `ownFocus`? 

This helps keyboard-only and screen-reader users navigate to and through the popover more quickly and easily. 

```suggestion
      <EuiPopover
        ownFocus
```